.htmlvalidateignore

```diff
@@ -0,0 +1 @@
+deploy/public/components/*/*/index.html
```

Comment:
Mind sticking a new line at the end here?

Usually an EditorConfig job!

Reply:
nice catch, thanks!

VS Code seems to be ignoring my .editorconfig file. I'll have to dig around and figure out why.